.eslintrc.js

```diff
@@ -1,61 +1,58 @@
 const path = require('path')
```

Comment:
Silly question, and it might be obvious given we aren't seeing any files be autoformatted in this PR, but I'm assuming that we're not changing style guidelines, and therefore none of the files in the codebase will be changed in the next PR that has these hooks?

Reply:
There's one minor style changes that we made to avoid escapes for quotes -- 
https://github.com/b12io/metronome/blob/1910962e0504aee7b08d2325e26320ecffc71791/.eslintrc.js#L41

Unfortunately, even though we have these lint and style rules, they are not enforced in our CI. We'll see quite a lot of changes in the subsequent PR where we fix all the styles and lints.

Reply:
That's great! Thank you. Given in our other repositories we lint in CI, should we also add lint enforcement in this repo?

Reply:
Sounds good, I'll add it as a github action job step.

Reply:
Looks amazing! Thank you! I love that the annotations appear in the code! :)